test/funnels/Depositor.t.sol

```diff
@@ -357,12 +356,14 @@ contract DepositorTest is DssTest, TestUtils {
         emit Withdraw(FACILITATOR, DAI, USDC, liquidity, withdrawn0, withdrawn1, fees0, fees1);
         vm.prank(FACILITATOR); depositor.withdraw(dp, true);
 
+        assertTrue(fees0 > 0 || fees1 > 0);
+        assertTrue(withdrawn0 > 0 || withdrawn1 > 0);
+        assertEq(GemLike(DAI).balanceOf(address(buffer)), prevDAI + withdrawn0 + fees0);
+        assertEq(GemLike(USDC).balanceOf(address(buffer)), prevUSDC + withdrawn1 + fees1);
         assertTrue(
             (fees0 > 0 && withdrawn0 > deposited0 && GemLike(DAI ).balanceOf(address(buffer)) > initialDAI ) ||
             (fees1 > 0 && withdrawn1 > deposited1 && GemLike(USDC).balanceOf(address(buffer)) > initialUSDC)
         );
```

Comment:
Not really related to this PR but I would get rid of that assertion as its success is too dependent on the particular way we generate fees in this test. In principle it's possible to run trades back and forth in such a way that fees0 > 0 and fees1 > 0 but leaving the pool in a state where withdrawn0 == deposited0 and withdrawn1 == deposited1. Besides, it would seem from reading that test that the difference `withdrawn{i} - deposited{i} == fees{i}` but that's not actually the case.

Reply:
Done.

Reply:
I'm merging, cc @oldchili so he is aware this check has been removed.